src/components/ChannelPlaylist.jsx

```diff
@@ -0,0 +1,33 @@
+import useStore from '../hooks/useStore';
+
+import ChannelCard from './ChannelCard';
+
+export default function ChannelPlaylist() {
+	const currentItem = useStore(state => state.currentItem);
+	const channels = useStore(state => state.channels);
+
+	const channelItems = channels?.map(channel => channel.snippet);
+
+	return (
+		<>
+			{currentItem.id === ''
+				? channelItems?.map(channelItem => {
+						return (
+							<ChannelCard key={channelItem.channelId} channelItem={channelItem} />
+						);
+				  })
```

Comment:
To avoid the empty string in your condition you can write my suggestion below - but that is just a picky note

```suggestion
			{!currentItem.id ? channelItems?.map(channelItem => {
						return (
							<ChannelCard key={channelItem.channelId} channelItem={channelItem} />
						);
				  })
```

Reply:
@JessicaLoers  Great suggestion! Thanks!

Reply:
@JessicaLoers  Actually it's not working that way, because then it is fetching also YouTube channels with the name `undefined`